docs/proposals/20230123-azure-service-operator.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Managing Azure Resources with Azure Service Operator
+authors:
+  - "@nojnhuh"
+reviewers:
+  - "@CecileRobertMichon"
+  - "@jackfrancis"
+  - "@matthchr"
+  - "@devigned"
+creation-date: 2023-01-23
+last-updated: 2023-02-21
+status: provisional
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/416
+  - "/docs/proposals/20210716-async-azure-resource-creation-deletion.md"
```

Comment:
should this be a link?

Reply:
I copied this syntax from the CAPI template: https://github.com/kubernetes-sigs/cluster-api/blob/323e257147a9a9094dde1215aada4e33a0788278/docs/proposals/YYYYMMDD-template.md?plain=1#L10-L12. Is there a better way to link to that existing proposal doc?